README.md

```diff
@@ -8,7 +8,70 @@ assisted close reading tool
 
 [https://github.com/ADAH-EviDENce/evidence-gui](https://github.com/ADAH-EviDENce/evidence-gui)
 
+## Running a demo on Windows
 
+Prerequisites:
+- Docker desktop
```

Comment:
Can you add a link?
https://docs.docker.com/docker-for-windows/install/

Reply:
good idea